pyproject.toml

```diff
@@ -54,3 +154,13 @@ src_paths = [
 line_length = 79
 skip_gitignore = true
 combine_as_imports = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--doctest-modules -s -vv --durations=0"
+doctest_optionflags = "NORMALIZE_WHITESPACE ELLIPSIS"
+junit_family = "xunit2"
+
+[tool.codespell]
```

Comment:
Note to using codespell on its own (not via precommit) requires an additional install of the `tomli` package.

https://github.com/codespell-project/codespell#using-a-config-file